local-cli/start_packager.sh

```diff
@@ -0,0 +1,22 @@
+
+set -e
+
+HEADER="${SRCROOT}/Modules/RCTIPAddress.h"
+
+echo "// This file is generated by a build script." > $HEADER
+echo "// Updated on "`date` >> $HEADER
+echo "" >> $HEADER
+
+ip_address="`ifconfig en0 | grep "inet " | cut -d: -f2 | awk '{print $2}'`"
```

Comment:
`ipconfig getifaddr en0` is less magical and will be easier to fix in the future. If we want to go the more complex route, we can use `ifconfig` command to find and store *all* available interfaces, and somehow offer the user to choose from the list. I don't think it's worth it at this point.